google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -13,6 +13,8 @@ language_settings:
     package_name: Google::Cloud::Speech::V1
   php:
     package_name: Google\Cloud\Speech\V1
+  nodejs:
+    package_name: "@google-cloud/speech"
```

Comment:
Could you file an issue to add support for the generating the node package in the config generator?